### PR TITLE
Integrate dennis linting for .po files.

### LINKED ISSUE #3336: Lint l10n translations with dennis-cmd
Part of https://github.com/mozilla/addons-frontend/issues/529

To avoid things like broken placeholders or other obvious problems.

* [ ] automate and document dennis-cmd installation
* [ ] Integrate dennis-cmd into linting package.json, potentially run it after `build-locales` script
* [ ] add to travis to run on every commit to catch these problems early on in the development process

### PR DESCRIPTION
**Pull request by EnTeQuAk**
Fixes #3336

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -29,5 +29,12 @@ commands = pytest tests/ui/test_discopane.py {posargs}
 deps = flake8==3.3.0
 commands = flake8 {posargs:.}
 
+[testenv:dennis-lint]
+deps = dennis==0.9
+# Run via `bash -c` to make sure the file globbing are properly resolved on the bash
+commands = bash -c 'dennis-cmd lint --errorsonly locale/*/LC_MESSAGES/*.po'
```
Comment (muffinresearch) [inline tox.ini]: This won't check the pot files? Should they be validated too? E.g. https://github.com/mozilla/addons-frontend/tree/master/locale/templates/LC_MESSAGES

File: tox.ini [hunk 2]
```diff
@@ -29,5 +29,12 @@ commands = pytest tests/ui/test_discopane.py {posargs}
 deps = flake8==3.3.0
 commands = flake8 {posargs:.}
 
+[testenv:dennis-lint]
+deps = dennis==0.9
+# Run via `bash -c` to make sure the file globbing are properly resolved on the bash
```
Comment (muffinresearch) [inline tox.ini]: s/file globbing are properly resolved on the bash/file globbing is properly resolved in the shell/

File: tox.ini (hunk 1 above)
Comment (EnTeQuAk) [reply tox.ini]: Fair point, we never validated the .pot files in olympia but might be a good idea. I'll add it here, won't hurt I guess.

Comment (muffinresearch) [reply tox.ini]: Yeah the .pot files should be good, but this should hopefully help us in a case where the extraction process does something odd.

**Comment by kumar303** [comment]
@EnTeQuAk thanks for adding this!

**Pull request merged**